src/lib/y2storage/used_storage_features.rb

```diff
@@ -112,8 +112,29 @@ class UsedStorageFeatures
     #----------------------------------------------------------------------
     #
 
-    def initialize(devicegraph)
-      @devicegraph = devicegraph
+    # Initialize object.
+    #
+    # @param arg [Integer or ::Storage::Devicegraph] Either a integer with the feature
```

Comment:
NP: I would say yardoc expects comma instead or *or* to separate different types: `@param arg [Integer, ::Storage::Devicegraph]`